assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -1288,6 +1288,10 @@ describe("DiversionPage", () => {
       fireEvent.click(originalRouteShape.get(container))
     })
 
+    await act(async () => {
```

Comment:
This is why I noticed [this](https://github.com/mbta/skate/pull/2745/files#r1725668252), because if this `await act` is moved down below the second `fireEvent.click`, then the test fails.